fern/src/vm.rs

```diff
@@ -57,19 +69,22 @@ impl Default for VM {
 }
 
 impl VM {
-    pub fn load(&mut self, program: &[Op]) -> VMResult {
+    pub fn load(&mut self, program: &[Op]) {
+        // TODO: should this be an assertion or a `VMError`?
+        assert!(
+            program.len() <= MAX_CODE_LENGTH,
+            "program too large to load (length={}, max length={})",
+            program.len(),
+            MAX_CODE_LENGTH
+        );
```

Comment:
I think this and errors decoding Ops should be VM errors, since reading and validating code is a VM responsibility. Thoughts?

Reply:
I feel like it's categorically incorrect to load a larger program than the VM can handle.

Reply:
I'm usually for giving the user the option instead of panicking when writing libraries, but tbh this is not really a library in the traditional sense so I think panic makes sense here.